packages/esm-patient-chart-app/src/ui-components/patient-chart.component.tsx

```diff
@@ -23,6 +23,10 @@ const PatientChart: React.FC<RouteComponentProps<PatientChartParams>> = ({ match
   const state = useMemo(() => ({ patient, patientUuid }), [patient, patientUuid]);
   const { windowSize, openWindows } = useContextWorkspace();
 
+  useEffect(() => {
+    !loading && detachAll('patient-chart-workspace-slot');
```

Comment:
Hm this is neither explanatory nor robust (for instance if we would switch to SWR and revalidate the data this would not work).

Why not close the workspaces when the patient chart is unmounted?